src/battle_game/core/units/double_scatter_tank.cpp

```diff
@@ -0,0 +1,196 @@
+#include "double_scatter_tank.h"
```

Comment:
Please use full path start from include directory.
Should be:
```suggestion
#include "battle_game/core/units/double_scatter_tank.h"
```